virtwho/config.py

```diff
@@ -1071,14 +1075,16 @@ def _validate_list(self, list_key):
             self._values[list_key] = []  # Reset to empty list
         return result
 
-    def add_key(self, key, validation_method=None, default=None, destination=None):
+    def add_key(self, key, validation_method=None, default=None, destination=None, required=False):
```

Comment:
This is good idea, but I would like to see some unit tests covering this new functionality.